gulpfile.js

```diff
@@ -91,37 +91,47 @@ function writeServiceWorkerFile() {
     cacheId: 'polymerproject',
     staticFileGlobs: [
       `${rootDir}/1.0/index.html`,
+      `${rootDir}/1.0/shell.html`,
       `${rootDir}/1.0/about.html`,
       `${rootDir}/elements/**`,
       `${rootDir}/js/*.js`,
       `${rootDir}/css/*.css`,
       `${rootDir}/bower_components/**/webcomponents-lite.min.js`,
     ],
-    navigateFallback: '/1.0/index.html',
+    dynamicUrlToDependencies: {
+      '/1.0/shell.html': [
+        path.join(rootDir, 'templates', 'base-devguide.html'),
+        path.join(rootDir, 'templates', 'head-meta.html'),
+        path.join(rootDir, 'templates', 'site-nav.html'),
+        path.join(rootDir, 'templates', 'base-blog.html'),
+        path.join(rootDir, 'templates', 'page-md-styles.html'),
+      ]
+    },
+    navigateFallback: '/1.0/shell.html',
```

Comment:
It exists in the dist/ folder, but if you change the path in `dynamicUrlToDependencies`, above (which eliminates an excess redirect), the SW will request and cache the file as `/1.0/shell`, so you should change it here, too.